db/seeds.rb

```diff
@@ -1,7 +1,42 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+# Data seeded into db
+Airport.destroy_all
+Flight.destroy_all
+
+airports_list = [
+  { name: "Murtala Muhammed International Airport", country: "Nigeria", city: "Lagos", code: "MF1" },
+  { name: "Nnamdi Azikwe International Airport", country: "Nigeria", city: "Lagos", code: "MF2" },
+  { name: "John Lennon Airport", country: "England", city: "Liverpool", code: "MF3" },
+  { name: "Manchester Airport", country: "England", city: "Manchester", code: "MF4" },
+  { name: "Gatwick Airport", country: "England", city: "West Sussex", code: "MF5" },
+  { name: "Heathrow Airport", country: "England", city: "London", code: "MF6" },
+  { name: "Los Angeles International Airport", country: "US", city: "Los Angeles", code: "MF7" },
+  { name: "San Francisco International Airport", country: "US", city: "San Francisco", code: "MF8" },
+  { name: "Indira Gandhi International Airport", country: "India", city: "Delhi", code: "MF9" },
+  { name: "Dubai International Airport", country: "Dubai", city: "Garhoud", code: "MF10" },
+  { name: "Beijing Capital International Airport", country: "China", city: "Beijing", code: "MF11" },
+  { name: "Singapore Changi Airport", country: "Singapore", city: "Changi", code: "MF13" },
+  { name: "Istanbul Atatürk Airport", country: "Turkey", city: "Istanbul", code: "MF14" },
+  { name: "Frankfurt Airport", country: "Germany", city: "Frankfurt", code: "MF15" },
+  { name: "Amsterdam Airport Schiphol", country: "Holland", city: "Haarlemmermeer", code: "MF16" }
+
+]
+
+airports_list.each { | airport | Airport.create(airport) }
+
+airlines = [ { "Qatar Airways" => 100 }, { "Singapore Airlines" => 90 }, { "Cathay Pacific Airways" => 80 }, { "Turkish Airlines" => 70 }, { "Emirates" => 60 }, { "Etihad Airways" => 50 }, { "Lufthansa" => 40 }, { "Air France" => 30 }, { "British Airways" => 20 }, { "KLM" => 10 } ]
+
+def set_flights(dist_in_hours, airports_list, airlines)
+  rand_airline = airlines.sample
+  airline = rand_airline.keys[0]
+  cost = dist_in_hours * rand_airline[airline] * 755
+  dept_time = Time.at(Time.now.to_i + (24 * 60 * 60 * generate_rand(1,21)))
```

Comment:
Space missing after comma.<br>Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.